tracing.go

```diff
@@ -34,6 +34,11 @@ import (
 	"golang.org/x/net/context"
 )
 
+// TracerProvider returns an OpenTracing Tracer
+type TracerProvider interface {
```

Comment:
well, the only reason is that it's used from json and thrift packages. But we could have a public helper method in tracing.go to extract the tracer from the registrar, wdyt?